pipeline/src/lib.rs

```diff
@@ -77,6 +77,7 @@ impl HostContext {
 // TODO at some point, we could also just pass evaluator::Values around - would be much faster.
 pub fn parse_query(query: &str) -> Result<(&str, Vec<&str>), String> {
     // We are expecting an enum value
+    let query = query.strip_prefix("std::prelude::Query::").unwrap_or(query);
```

Comment:
Is this change needed, and is so, why?

Reply:
Because enum values are now printed properly with the enum name: https://github.com/powdr-labs/powdr/pull/1799/files#diff-f00565b83aa549438943c22a062d9bd3e6c27f77e39d2dcb6edbf456aa53e76aR226

I.e. before this, it would just print `Input`, now it prints `std::prelude::Query::Input`